src/main/kotlin/no/nav/syfo/fastlege/FastlegeService.kt

```diff
@@ -66,7 +66,7 @@ class FastlegeService(
 
     fun hentBehandlereForKontor(
```

Comment:
Burde denne få nytt navn nå som vi henter "kontor" i stedet for "behandlere på kontor"? 🤔

Reply:
Ja, kanskje det - skal tenke på det.